src/components/Modal/Modal.test.tsx

```diff
@@ -24,14 +16,16 @@ describe("Modal ", () => {
         Button tester
       </Modal>
     );
-    expect(wrapper).toMatchSnapshot();
+    expect(wrapper.find(".p-modal__footer button").length).toEqual(2);
   });
 
   it("displays any content in the button row", () => {
     const wrapper = shallow(
       <Modal buttonRow={<div>I am not a button</div>}>Button tester</Modal>
     );
-    expect(wrapper).toMatchSnapshot();
+    expect(wrapper.text()).toContain("I am not a button");
+    expect(wrapper.text()).toContain("Button tester");
+    expect(wrapper.text()).not.toContain("I am something else");
```

Comment:
I'm not entirely sure but I think you've added a 'falsy' test case to ensure that the `toContain` assertions aren't just arbitrarily passing. Laudable goal but in this case it's not really necessary as you're effectively testing that the assertion library is working....and well....if it's not we have bigger problems 😆 

Where you'd typically want to do this is if the Modal would have the string "I am something else" in the event that nothing was defined with `buttonRow`. Then you'd want to make sure it didn't exist vs just being appended to.